packages/react/src/Tooltip/Tooltip.features.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import type {Meta} from '@storybook/react'
+import {Button, Box, Link, Stack} from '..'
+import {Tooltip} from './Tooltip'
+import {BookIcon} from '@primer/octicons-react'
+
+export default {
+  title: 'Components/Tooltip/Features',
+  component: Tooltip,
+} as Meta<typeof Tooltip>
+
+export const AnchorHasMargin = () => (
+  <Box padding="spacious">
+    <Tooltip text="Tooltip is still centered">
+      <Button sx={{marginLeft: 3}}>Button has 16px margin Left</Button>
+    </Tooltip>
+  </Box>
+)
+
+export const LabelType = () => (
+  <Box padding="spacious">
+    <Tooltip text="Contribution Documentation for 'Primer React'" type="label">
+      <Link href="https://github.com/primer/react/contributor-docs/CONTRIBUTING.md">
+        <BookIcon />
+      </Link>
+    </Tooltip>
+  </Box>
+)
+
+// As a supplementary description for a button
+export const DescriptionType = () => (
+  <Box padding="spacious">
+    <Tooltip text="Supplementary text" direction="n">
+      <Button>Save</Button>
+    </Tooltip>
+  </Box>
+)
```

Comment:
This example looks the same as the default example, but with an alternative direction. Is that right? Wondering if we can skip this one and dedupe? WDYT @mperrotti, i'm probably missing something crucial here though.

Reply:
You're not missing anything critical. Since "description" tooltips are the default, this is essentially the same as the default. Happy to remove.